File: 2022/01/go/1a.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func ona_a() {
	readFile, err := os.Open("../input.txt")
	total := 0
	highestTotal := 0

	if err != nil {
		fmt.Println(err)
	}

	fileScanner := bufio.NewScanner(readFile)

	fileScanner.Split(bufio.ScanLines)

	for fileScanner.Scan() {
		currentLine := fileScanner.Text()
		if currentLine == "" {
			if total > highestTotal {
				highestTotal = total
			}
			total = 0
		} else {
			currentLineValue, _ := strconv.Atoi(currentLine)
			total += currentLineValue
		}
	}

	readFile.Close()
	fmt.Println(highestTotal)
}

File: 2022/01/python/1a.py

def find_highest_calories():
    total = 0
    highest_total = 0
    with open("../input.txt") as f:
        for line in f:
            if line == "\n":
                highest_total = total if total > highest_total else highest_total
                total = 0
            else:
                total += int(line)
    return highest_total

print(find_highest_calories()) # 67658